### DISCUSSION TITLE
Can't find RemoveBorderEffect under MAUI Community toolkit package

### DISCUSSION THREAD

**Discussion by KDewangan** [CommunityToolkit/Maui#1035] [Q&A]
Hi Everyone, 
I am migrating a Xamarin.Forms project to .NET MAUI, we have used RemoveBorderEffect from Xamarin Community toolkit quite a lot in the project, And I was thinking that MAUI Community toolkit will have the RemoveBorderEffect, but I couldn't see that under MAUI Community toolkit package. Is it available in the MAUI Community toolkit? Or is there any other effect that does the same function?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
